src/content/en/2019/markup.md

```diff
@@ -0,0 +1,301 @@
+
+Note how quickly use drops off.
+
+11 elements occur in over 90% `<html>`, `<head>`, `<body>`, `<title>`, `<meta>`, `<a>`,`<div>`, `<link>`, `<script>`, `<img>` and `<span>`. 
+
+Only 15 more *elements occur in at least 50% of the home pages (`<ul>`, `<li>`, `<p>`, `<style>`, `<input>`, `<br>`, `<form>`, `<h2>`, `<h1>`, `<iframe>`, `<h3>`, `<button>`, `<footer>`, `<header>`, `<nav>` are the others).  
+
+And only 40 more elements occur on more than 5% of pages.
+
+Even `<video>`, for example, doesn't make that cut.  It appears on only 4.21% of pages in the dataset (on desktop, only 3.03% on mobile).  While these numbers sound very low, 4.21% is actually *quite* popular by comparison.  In fact, only 98 elements occur on more than 1% of pages.  
+
+It's interesting, then, to look at what the distribution of these elements looks like and which ones have more than 1% use.  Below is a chart that shows the rank of each element and which category they fall into.  I've separated the data points into discrete sets simply so that they can be viewed (otherwise there just aren't enough pixels to capture all that data), but they represent a single 'line' of popularity - the left-most being the most common, the right-most being the least common.  The arrow points to the end of elements that appear in more than 1% of the pages.
+
+(( TODO: there is a corresponding image in the google doc https://docs.google.com/document/d/16TY_pV-FyW35DzuvdlaOiENz6o6PWYpl_RviU-HW7Qc/edit ))
+
+You can observe two things here: First, that the set of elements that have more than 1% use are not exclusively HTML.  In fact, *27 of the most popular 100 elements* *aren't even HTML *- they are SVG! And there are *non-standard tags at or very near that cutoff too*!  Second, note that a whole lot of HTML elements are used by less than 1% of pages.
+
+ 
+
+So, are all of those elements used by less than 1% of pages "useless?".  Definitely not.  This is why establishing perspective matters.
+
+`<code>`, for example, is an element that I both use and encounter a lot.  It's definitely important - and yet it is used on only 0.57% of these pages.  Part of this is skewed based on what we are measuring - home pages are generally* less likely* to include certain kinds of things (like `<code>` for example): They serve a less general purpose than, for example, headings, paragraphs, links and lists, however, the data is generally useful.
+
+We also collected information about which pages contained an author defined (not native) .shadowRoot - About 0.22% of the pages on desktop, and 0.15% on mobile.  This might not sound like a lot, but it is roughly 6.5k sites in the mobile dataset and 10k sites on the desktop and is more than several HTML elements.  `<summary>` for example, has about equivalent use on the desktop and it is the 146th most popular element.
```

Comment:
```suggestion
We also collected information about which pages contained an author defined (not native) `.shadowRoot` -- About 0.22% of the pages on desktop, and 0.15% on mobile.  This might not sound like a lot, but it is roughly 6.5k sites in the mobile dataset and 10k sites on the desktop and is more than several HTML elements.  `<summary>` for example, has about equivalent use on the desktop and it is the 146th most popular element.
```